packages/pwa-buildpack/lib/WebpackTools/MagentoResolver.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
     async configure(options) {
         validateConfig('.configure()', options);
         return {
+            alias: {},
             modules: [options.paths.root, 'node_modules'],
             mainFiles: ['index'],
-            extensions: ['.mjs', '.js', '.json', '.graphql']
+            mainFields: ['esnext', 'es2015', 'module', 'browser', 'main'],
+            extensions: ['.wasm', '.mjs', '.js', '.json', '.graphql']
```

Comment:
IIRC, one optimization to make webpack resolve a lot faster is to cut this field down to just `[.js]`, precluding the need to [resolve them in order](https://webpack.js.org/configuration/resolve/#resolveextensions). I'm surprised we aren't already doing it, actually; could we do it here?

Not only other the other extensions rarely imported, we already tend to include them explicitly for clarity's sake.

```es6
// safe to assume `./foo` refers to `foo.js` or `foo/index.js`
import Foo from "./foo"

// by explicitly appending `.css`, we avoid ambiguity or surprise
import defaultClasses from "./foo.css"
```

Reply:
I'm not seeing that advantage reliably when I try that. With no cache it's always around 50 seconds, and with full cache it's always around 20.